portfolio/src/main/webapp/index.html

```diff
@@ -9,11 +9,17 @@
   <body>
     <div id="content">
       <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
+      <p>Carine Gordillo</p>
+      <p> Hello! I'm Carine  and I live in Downey, California but parents are originally from<br/> Zapopan, Jalisco. Some of my favorite hobbies are hiking, being with my family and
+      <br/> taking on new projects. I am 19 years old and attend Cal State Long Beach. Currently, I am <br/> participating in a progam by Google called SPS and I am on team 41! </p>
```

Comment:
At Google, we have a guideline around line length, where your code shouldn't exceed 80 characters on any given line (generally, there are some exceptions). GitHub doesn't show how many characters each line is, but I have to scroll to see it all, so I would recommend splitting this text up into more lines. Maybe end around "Some of my favorite hobbies are [new line] hiking, being..." and following that roughy for future commits as well.